doc/guides/collaborator-guide.md

```diff
@@ -120,14 +120,22 @@ needed [approvals](#code-reviews), [CI](#testing-and-ci), and
 except the [wait time](#waiting-for-approvals), please add the
 [`author ready`](#author-ready-pull-requests) label.
 
-Where there is disagreement among Collaborators, consensus should be sought if
-possible. If reaching consensus is not possible, a Collaborator may escalate the
-issue to the TSC.
-
-Collaborators should not block a pull request without providing a reason.
-Another Collaborator may ask an objecting Collaborator to explain their
-objection. If the objector is unresponsive, another Collaborator may dismiss the
-objection.
+If a collaborator believes a pull request should not land as is, **the "Request
+Changes" GitHub feature must be used to make the objection explicit**. An
+implicit objection not using the "Request Changes" feature is not considered a
+blocker for a pull request. Pull requests with an explicit objection should
```

Comment:
Both of these two sentenences seem to be restatements of the first statement. I appreciate making sure it is clear, but perhaps just a single restatement would be fine between these?

To add another variation to the mix, another restatement could be a comprehensive statement of the positive criteria for landing a PR.

> A PR may land only when both the wait time has passed and there are no explicit objections as marked through the GitHub "request changes" feature.

Reply:
I'll try to rephrase it so it's less repetitive.

Reply:
Sorry, forgot to address these before landing. Sent another PR that should reduce repetitiveness: https://github.com/nodejs/node/pull/34702